README.md

```diff
@@ -1 +1,36 @@
-# Conference
+# Marketing Conference
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+Created a global marketing conference website. It has Home Page and About Page and also added the user interaction link, mobile menu, Implement the section "Featured speakers" with dynamic HTML.
+In mobile version when 'MORE' button is dynamically used and followed the design.
+
+
+
+## Built With
+
+- HTML & CSS
+- Java Script
+- linters
+
```

Comment:
Kindly record a video presenting the features of the project with:
- Maximum length of the video is 5 minutes.
- The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
- The video contains a demo of the project, showing the pages in different screen sizes.
- The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

and add it's link into the README file.

Reply:
It is not Required as per the description